cvcl/migrations/0001_initial.py

```diff
@@ -59,10 +59,10 @@ class Migration(migrations.Migration):
                 ('username', models.CharField(max_length=30)),
                 ('limit_instances', models.IntegerField(default=0)),
                 ('limit_cpus', models.IntegerField(default=0)),
-                ('limit_ram', models.IntegerField(default=0, verbose_name='RAM limit in bytes')),
+                ('limit_ram', models.IntegerField(default=0, verbose_name=b'RAM limit in bytes')),
```

Comment:
https://stackoverflow.com/questions/6269765/what-does-the-b-character-do-in-front-of-a-string-literal